eclair-core/src/test/scala/fr/acinq/eclair/integration/IntegrationSpec.scala

```diff
@@ -261,7 +261,7 @@ class IntegrationSpec extends TestKit(ActorSystem("test")) with BitcoindService
     val pr = sender.expectMsgType[PaymentRequest]
     // then we make the actual payment
     sender.send(nodes("A").paymentInitiator,
-      SendPayment(amountMsat.amount, pr.paymentHash, nodes("D").nodeParams.nodeId, routeParams = integrationTestRouteParams))
+      SendPayment(amountMsat.amount, pr.paymentHash, nodes("D").nodeParams.nodeId, routeParams = integrationTestRouteParams, maxAttempts = 5))
```

Comment:
Although that was the previous behavior, there is no need to retry payments up to five times in integration test. Just tried locally and it works fine setting them to 1.